experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -222,11 +222,11 @@
                 }
             ],
             "elseActions": [
-                ${actionSetOrPush(@datetime.timex[0])}
+                ${actionSetOrPush('@datetime.timex[0]')}
```

Comment:
Do we need quotes？

Reply:
Yes.  Otherwise this is evaluated at generation time.